compiler/rustc_codegen_llvm/src/debuginfo/metadata/enums/mod.rs

```diff
@@ -55,6 +54,14 @@ pub(super) fn build_enum_type_di_node<'ll, 'tcx>(
 
     let enum_type_and_layout = cx.layout_of(enum_type);
 
+    if cpp_like_debuginfo(cx.tcx)
+        && tag_base_type_opt(cx.tcx, enum_type_and_layout)
+            .map(|ty| ty.primitive_size(cx.tcx).bits())
+            == Some(128)
```

Comment:
`tag_base_type_opt(cx.tcx, enum_type_and_layout).map(|ty| ty.primitive_size(cx.tcx).bits()) == Some(128)` seems to be repeated a few times. Maybe pull it into a helper function?